### PR TITLE
CLOUDP-216773: Tweak Atlas CLI for Atlas Stream Processing

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-216773

Small copy fix

### CODE REVIEW

**Review by blva — APPROVED** [review]

File: docs/atlascli/command/atlas-streams.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ atlas streams
    :depth: 1
    :class: singlecol
 
-Manage streams for your project.
+Manage your Atlas Stream Processing Deployments.
```
Comment (erabil-mdb) [inline docs/atlascli/command/atlas-streams.txt:15]:
```suggestion
Manage your Atlas Stream Processing deployments.
```

File: docs/atlascli/command/atlas.txt [hunk 2]
```diff
@@ -81,7 +81,7 @@ Related Commands
 * :ref:`atlas-security` - Manage security configuration for your project.
 * :ref:`atlas-serverless` - Manage serverless instances for your project.
 * :ref:`atlas-setup` - Register, authenticate, create, and access an Atlas cluster.
-* :ref:`atlas-streams` - Manage streams for your project.
+* :ref:`atlas-streams` - Manage your Atlas Stream Processing Deployments.
```
Comment (erabil-mdb) [inline docs/atlascli/command/atlas.txt:84]:
```suggestion
* :ref:`atlas-streams` - Manage your Atlas Stream Processing deployments.
```

File: internal/cli/atlas/streams/streams.go [hunk 3]
```diff
@@ -26,10 +26,13 @@ func Builder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     use,
 		Aliases: cli.GenerateAliases(use),
-		Short:   "Manage streams for your project.",
-		Long:    `The streams command provides access to your Atlas Stream Processing configurations. You can create, edit, and delete streams, as well as change the connection registry.`,
+		Short:   "Manage your Atlas Stream Processing Deployments.",
```
Comment (erabil-mdb) [inline internal/cli/atlas/streams/streams.go:29]:
```suggestion
		Short:   "Manage your Atlas Stream Processing deployments.",
```

**Review by erabil-mdb — APPROVED** [review]
LGTM % one nit.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**